main/integration-tests/__test__/api-tests/common/data-sources/register-ds-study.test.js

```diff
@@ -13,8 +13,8 @@
  *  permissions and limitations under the License.
  */
 
-const { runSetup } = require('../../../support/setup');
-const errorCode = require('../../../support/utils/error-code');
+const { runSetup } = require('../../../../support/setup');
+const errorCode = require('../../../../support/utils/error-code');
 
 describe('Register data source study scenarios', () => {
```

Comment:
We seem to have the same filename `register-ds-study.test.js` in both common and non-appstream path

Reply:
Ah yes. Each of the two files is a subset of the original `register-ds-study.test.js`. The test cases that passes for the AppStream enabled path is in the common folder. The test cases that doesn't pass for AppStream enabled is in the `appstream-disabled` folder.

Reply:
`should return study registration information if admin`fails because it trying to [create Studies](https://github.com/awslabs/service-workbench-on-aws/blob/b799b5bfb1305c07b04742725697c6b0e5f47bdc/main/integration-tests/support/resources/data-sources/studies.js#L50) with `accessType=readWrite` which is not allowed by AppStream [registerStudy code](https://github.com/awslabs/service-workbench-on-aws/blob/64c67609961295a343f5b90a859ff2046ee2c44e/addons/addon-base-raas/packages/base-raas-services/lib/study/study-service.js#L199). We don't allow that because if a user can create a study in which they have write access then they'll be able to export data without using the `EgressDataStore`.

Reply:
Ah, so it is an egress store setting as opposed to AppStream setting. I guess we should clarify that with folder structure then. Something like `appstream-and-egress-enabled` perhaps?

Reply:
good point, I'll change the folder name to reflect that. I'll use `appstream-egress-enabled` to keep the name shorter